### PR TITLE
feat(chrome): add configurable timeout status code

### PR DESCRIPTION
**Pull request by Golyo88**
Closes: 418/1

### CODE REVIEW

File: test/browsers/chrome-spec.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const Sinon = require('sinon');
+var chrome = require('../../lib/browsers/chrome'),
```
Comment (varrocs) [inline test/browsers/chrome-spec.js:2]: var -> const

File: test/browsers/chrome-spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    sinon = require('sinon'),
+    assert = require('assert')
+
+describe('chrome', function() {
+    describe('loadUrlThenWaitForPageLoadEvent', function() {
+        var tab;
```
Comment (varrocs) [inline test/browsers/chrome-spec.js:8]: var -> let

File: test/browsers/chrome-spec.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+		afterEach(function() {
+			sandbox.restore();
+		});
+
+        it ('Should NOT change tabs status code', async function() {
+            const expectedStatusCode = 123;
+            tab.prerender.statusCode = expectedStatusCode;
+
+            const p = chrome.loadUrlThenWaitForPageLoadEvent(tab, 'the-url');
```
Comment (varrocs) [inline test/browsers/chrome-spec.js:37]: Is there a reason that these are in separate lines?

File: test/browsers/chrome-spec.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+var chrome = require('../../lib/browsers/chrome'),
+    sinon = require('sinon'),
+    assert = require('assert')
+
+describe('chrome', function() {
+    describe('loadUrlThenWaitForPageLoadEvent', function() {
+        var tab;
+
+        beforeEach(function() {
+			sandbox = sinon.createSandbox();
```
Comment (varrocs) [inline test/browsers/chrome-spec.js:11]: const?

File: test/browsers/chrome-spec.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            tab.Page = sandbox.stub();
+            tab.Page.enable = sandbox.stub();
+            tab.Page.enable.resolves(1);
+            tab.Page.addScriptToEvaluateOnNewDocument = sandbox.stub();
+            tab.Page.navigate = sandbox.stub();
+            tab.Page.navigate.resolves(1);
+
+            tab.Emulation = sandbox.stub();
+            tab.Emulation.setDeviceMetricsOverride = sandbox.stub();
+		});
```
Comment (varrocs) [inline test/browsers/chrome-spec.js:27]: tabs vs spaces
some lines are intended with tabs while others with spaces

**Comment by Golyo88** [comment]
ch418

**Comment by Golyo88** [comment]
[ch418]

**Comment by Golyo88** [comment]
[ch418]

**Review by lluczo — APPROVED** [review]

**Pull request merged**